test/unit/range/view/view_translate_test.cpp

```diff
@@ -156,7 +156,7 @@ TYPED_TEST(nucleotide, view_translate)
 
     // combinability and function syntax
     auto v11 = detail::view_translate(view::complement(vec), translation_frames::FWD_REV_0);
-    // == [[T,Y,V,R],[Y,V,R,T]]
+    // == [[C,M,H,A],[M,H,A,C]]
```

Comment:
really? 🙈

Reply:
Yes, it is the same as ```v9``` as first the sequence complement is computed and then translated. Or am I missing something?

Reply:
Could well be, I didnt double check. It reminds me that we need to add infrstructure to test the output of snippets.

Reply:
It is at least compared to the vector that consists of these exact frames so since the test runs through it seems to be correct. Also this is not a snippet but a test case, right?

Reply:
My whole comment was just meant as "In general, maybe there should be something that checks the actual values"!
It was not meant as criticism of the change! I totally believe you that you checked the output.

Reply:
Then I agree :wink: